app/dashboard/migrations/0049_merge_20190823_0222.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.2.3 on 2019-08-23 02:22
+
+from django.db import migrations
```

Comment:
Im worried about the amount of migrations, maybe we will need to recreate it ??

Reply:
i can recreate them if we want, repo syncing to a repo model was added after the work was originally complete.